packages/monosize-storage-azure/src/types.mts

```diff
@@ -1,3 +1,9 @@
+export type AzureAuthenticationType = 'AzureNamedKeyCredential' | 'AzurePipelinesCredential';
```

Comment:
There are [a lot](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/samples/AzureIdentityExamples.md) of other identity authentication methods so this could be extended in the future

Reply:
It uses [Workload Identity](https://learn.microsoft.com/en-us/entra/workload-id/workload-identities-overview) which is how all our ADO pipelines authenticate with our azure subscription resources now. It's essentially using a workload identity federation [service connection](https://dev.azure.com/uifabric/fabricpublic/_settings/adminservices) which has explicit access to the azure resources